addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -43,7 +43,8 @@ $o-wsale-products-layout-grid-gutter-width: min($grid-gutter-width / 2, $o-wsale
 }
 
 @mixin o-wsale-scrollbar-subdle($-vertical: true) {
-    scrollbar-color: currentColor $light;
+    $-o-scrollbar-subdle-bg: rgba($dark, 0.05);
+    scrollbar-color: currentColor $-o-scrollbar-subdle-bg;
```

Comment:
@stefanorigano Could you take a look at this change to see if it suits you?
Thank you :pray: